common/build.gradle

```diff
@@ -1,9 +1,26 @@
+buildscript {
```

Comment:
Guessing you discussed this with @Prouser123 - gets the okay from me if he approves.